app/views/admin/versions/_object_desc_and_link.html.haml

```diff
@@ -9,13 +9,13 @@
 
 - when 'UsersRole'
   - users_role = current_or_last_object_state(version.item_type, version.item_id)
-  - role = Role.find_by(id: users_role.role_id) if users_role
+  - role = Role.find_by(id: users_role.role_id) || PaperTrail::Version.where(item_type: 'Role', item_id: version.item_id).where(id: users_role.role_id)
```

Comment:
version.item_id is the ID of the UsersRole record, not the id of a Role record. 

Why not find role like this:
`- role = current_or_last_object_state('Role', users_role.role_id)`

Reply:
And for name
- role_name = role.try(:name) || PaperTrail::Version.where(item_type: 'Role', item_id: users_role.role_id).last.changeset[:name].second

So that we can ensure showing something.